pkg/infra/tracing/tracing.go

```diff
@@ -47,21 +53,26 @@ func (ts *TracingService) Init() error {
 
 func (ts *TracingService) parseSettings() {
 	var section, err = ts.Cfg.Raw.GetSection("tracing.jaeger")
-	if err != nil {
-		return
+	if err == nil {
```

Comment:
You should always expect the `tracing.jaeger` section to be there. That's why we have the defaults.ini file of all sections and parameters. So I think if there's an error here we should return early. Or is there any other reason for doing this that I don't see?

Reply:
First - tests (I use an empty config there). Also, nothing prevents user from deleting that section and using environment variables only. Or I misunderstand it?

Reply:
If a user deletes that section from the defaults.ini file then the user is on their own basically. Grafana is always shipped with defaults.ini file and users can override that with a custom .ini file or environment variables.

Reply:
Yeah, I see in a few other packages sections are treated as if they always exist.
Should a missing `[tracing.jaeger]` section be considered an error then?

Reply:
I would say definitely yes. That it doesn't do that currently feels wrong.